src/eval-frame/actions/language-actions.js

```diff
@@ -101,62 +92,3 @@ export function loadLanguagePlugin(pluginData) {
   }
   return languagePluginPromise;
 }
-
-export function evaluateLanguagePlugin(pluginText, evalId) {
```

Comment:
this logic (parsing etc) how handled in saga on editor side; we just need loadLanguagePlugin now